File: c-obj/hello.m
#import "helloWorld.h"

@implementation Hello
- (void) sayHello {
	NSLog(_(@"Hello, world!"));
	NSLog("\n");
}
@end

int main(int argc, char *argv[], char *envp[], char *apple[]) {
    @autoreleasepool {
        [[Hello new] sayHello];
    }
    return 0;
}

File: c/hello.c
#include <stdio.h>
#include <libintl.h>
#include <locale.h>

#define _(String) gettext (String)

int main(int argc, char* argv[], char* envp[]) {
	printf(_("Hello, world!"));
	printf("\n");
	return 0;
}
